### PR TITLE
Added styling to error pages

### LINKED ISSUE #66: Apply style to error pages

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?
Closes #66 

#### What's this PR do?
Adds styling to error pages (404, 500)

#### How should this be manually tested?
The patch in this gist will set `DEBUG=False` in your `docker-compose.yml` and will set `/pay` to trigger a 500 error: https://gist.github.com/gsidebo/4c28f0faf31d69bce26723c0775c78e1. The 500 error page can be seen there. Testing the 404 page is simple enough as navigating to a bogus URL with `DEBUG=False` (again, you'll need to set that in your `docker-compose.yml`)

#### Any background context you want to provide?
We had/have several bugs regarding the use of `DEBUG=False` in a local env. One of them was fixed in `render_bundle.py` as part of this PR. Other bugs:

1. `DEBUG` cannot be set to true in `.env`. You need to do it in `docker-compose.yml`
1. Background images result in a 404 when `DEBUG=False`

#### Screenshots (if appropriate)
ss 2017-05-19 at 16 50 45
ss 2017-05-19 at 16 50 59

### CODE REVIEW

File: bootcamp/templates/404.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-<div id="page-not-found">
-	<p>Page not found</p>
-	<p>Whatever you were looking for, isn't here</p>
-	<p>Please <a href="mailto:{{ support_email }}">contact us</a> if this continues to happen</p>
+{% block content %}
+{% include "navbar.html" %}
+<div id="error" class="text-info-page">
+	<h2>Page not found</h2>
+	<p>Whatever you were looking for isn't here.</p>
+	<p>Please <a href="mailto:{{ support_email }}">contact us</a> if you need support.</p>
```
Comment (amir-qayyum-khan) [inline bootcamp/templates/404.html]: please use spaces for indentation

File: bootcamp/templates/500.html [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
-{% endblock %}
+{% block title %}MIT Bootcamps - Error{% endblock %}
 
-{% block error_content %}
-<div id="page-not-found">
-	<p>There has been an error (500).</p>
+{% block content %}
+{% include "navbar.html" %}
+<div id="error" class="text-info-page">
+	<h2>There has been an error (500).</h2>
```
Comment (amir-qayyum-khan) [inline bootcamp/templates/500.html]: same here

**Comment by amir-qayyum-khan** [comment]
@roberthouse54 any thoughts on UI/UX

**Comment by roberthouse54** [comment]
I don't know if we want to do this as a separate issue, but can we add a bit of styling and a background image to the error page(s)?

image

**Comment by gsidebo** [comment]
@roberthouse54 should be easy. send me the image over slack and i'll add it

**Comment by gsidebo** [comment]
@roberthouse54 what do you think? this matches the styling of the login page

ss 2017-05-23 at 15 30 18

**Comment by gsidebo** [comment]
@amir-qayyum-khan ready for review again

**Comment by roberthouse54** [comment]
👍

**Comment by gsidebo** [comment]
@amir-qayyum-khan due to some issues we have with the `DEBUG` setting in bootcamp and micromasters, you wont be able to access the error pages and also see background images. the patch in [this gist](https://gist.github.com/gsidebo/5d036f1e28149cc5f2c96273ea7e27bd) will add the urls `/404` and `/500` to the app so you can access them directly

**Pull request merged**